.github/workflows/BuildPR.yml

```diff
@@ -28,4 +27,11 @@ jobs:
       - name: pre build
         run: cd External/objective-git && script/bootstrap && script/update_libgit2 && cd ../..
       - name: Build project
-        run: xcodebuild -workspace GitX.xcworkspace -scheme GitX archive
+        run: |
+          xcodebuild -workspace GitX.xcworkspace -scheme GitX -archivePath ./GitX archive
+          tar -acf GitX.xcarchive.zip GitX.xcarchive
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
```

Comment:
What's about to avoid size by skipping failed jobs ?

https://github.com/hannesa2/mediapipe/blob/APK-0.8.2/.github/workflows/CI.yml#L44
I mean similar with
`if: ${{ success() }}`

Reply:
OK, I had assumed that, if the build failed, follow up jobs would be cancelled. This is nice and clear, though.